lib/install/validate/config.go

```diff
@@ -557,3 +560,52 @@ func (v *Validator) CheckDrs(ctx context.Context) {
 	log.Info("DRS check OK on:")
 	log.Infof("  %q", v.Session.Cluster.InventoryPath)
 }
+
+// check that PersistNetworkBacking is set
+func (v *Validator) CheckPersistNetworkBacking(ctx context.Context) {
+	errMsg := "vCenter settings check SKIPPED"
+	if !v.sessionValid(errMsg) {
+		return
+	}
+	if !v.IsVC() {
+		log.Info(errMsg)
+		return
+	}
+
+	val, err := optmanager.QueryOptionValue(ctx, v.Session, persistNetworkBackingKey)
+	if err != nil {
+		// the key is not set, set it now
+		val = "false"
+	}
+	var ok bool
+	if val != "true" {
+		ok = v.SetPersistNetworkBacking(ctx)
+		if !ok {
+			log.Errorf("vCenter settings check FAILED")
```

Comment:
This should be `v.NoteIssue` and should note user remediation, e.g. `requires vCenter advanced option vpxd.SerialPort.PersistNetworkBacking=true`.

Reply:
fixed